### DISCUSSION TITLE
Adding New SN Nodes (Peers)

### DISCUSSION THREAD

**Discussion by h-ahmad** [HewlettPackard/swarm-learning#111] [General]
I have successfully run the example with 2 peers as given here. I want to add one SN node; another SN node (peer). What are the changes I have to do? Thanks.

**Comment by iArpanPatel** [comment]
New SN node can be added to the network (--network) and be pointed to the sentinel SN (--sentinel-ip),
the same way as it has been done while creating SN2.

Moving this to discussion.

**Reply by h-ahmad** [reply]
What about the swop profiles?
What about the user, user/app-data folders?
What about the increase in PEERS from 2 to 3 in swci?
What about the certificates and sharing among peers?
Any other change required?
